lms/templates/public_video_share_embed.html

```diff
@@ -0,0 +1,106 @@
+## coding=utf-8
+
+<%page expression_filter="h"/>
```

Comment:
This was a Frankenstein of a page. The public video view uses `courseware-chromeless.html` which inherits from `main.html` our base lms template. 

This page was created by merging `courseware-chromeless` down into `main` and then deleting everything I could while still leaving the page functioning.

Some stuff I left: 
- Styling for the video player
- Internationalization that applies to the player (I figure RTL languages probably flip the forward slash of "time elapsed"/ "time remaining"? Probably other stuff I'd never think of? Best to leave it be.
- js that made the video break when i took it out
- page tracking, like segment and GA
- branch-io?
- google site verification?